testing/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Philipp Oppermann <dev@phil-opp.com>"]
 edition = "2018"
 
 [[test]]
-name = "breakpoint_exception"
+name = "interrupt_handling"
```

Comment:
This also isn't part of master yet. Could you undo this change and go back to the original `testing/tests/breakpoint_exception.rs` file as well?